### DISCUSSION TITLE
multiselect in cascade dropdown

### DISCUSSION THREAD

**Discussion by huining112** [Budibase/budibase#6274] [Help]
Hi there, i am building a cascade dropdown for users to try different neurovascular catheters and microcathethers combination, and i have set a budibase table with "surgery" text field to indicate different surgery senario, "catheter" text field to choose catheter to use, "microcatheter" multi-select field to choose microcatheters ( you can choose one catheter, and multiple microcatheters in one surgery type)  and whether the combination would works with remarks.

![image](https://user-images.githubusercontent.com/107283360/173161568-b8627920-9992-4a6b-aad1-07a397732a52.png)

when i built the screen, i use multi-select picker for the microcatheter field, with data provider filtered by surgery and catheter selected in form

![image](https://user-images.githubusercontent.com/107283360/173161665-44d4e931-6d41-4cdd-8776-00018566df09.png)

but the choices in dropdown are in the format of "microcatheter A, microcatheter B" rather than A and B appears as two seperate choice, which i can select at the same time. 

![image](https://user-images.githubusercontent.com/107283360/173162022-0ee2f4d1-0a6f-4a74-83c5-262638bb53d2.png)

how can i achieve this? thanks for guidance in advance

**Comment by melohagan** [comment]
Hi @huining112 

I think you need to use the **Relationship Picker** instead of the **Multi-select picker**

![Screenshot 2022-06-13 at 10 17 20](https://user-images.githubusercontent.com/101575380/173321588-af157a58-0a64-45b3-9f87-1cc72d8efa8f.png)

**Reply by huining112** [reply]
thanks, i will give a try!
